client/src/components/DataFiles/DataFilesSearchbar/DataFilesSearchbar.js

```diff
@@ -3,27 +3,28 @@ import PropTypes from 'prop-types';
 import { useHistory, useLocation } from 'react-router-dom';
 import { Button } from 'reactstrap';
 import queryString from 'query-string';
-import { Icon } from '_common';
+import { Icon, DropdownSelector } from '_common';
 import { findSystemDisplayName } from 'utils/systems';
 import { useSelector } from 'react-redux';
+import fileTypes from './FileTypes';
 
-import './DataFilesSearchbar.module.css';
+import './DataFilesSearchbar.module.scss';
 
 const DataFilesSearchbar = ({
   api,
   scheme,
   system,
+  setFilterType,
+  filterType,
   resultCount,
   className,
-  publicData
+  publicData,
+  siteSearch,
+  disabled
```

Comment:
When clicking on `Google Drive` in `Data Files`(Google account not linked), it seems to give me this error:
`Warning: Failed prop type: Invalid prop disabled of type string supplied to Button, expected boolean.`

Reply:
Good catch!